server/routes/serviceProviderReferrals/endOfServiceReportFormPresenter.ts

```diff
@@ -0,0 +1,44 @@
+import { SentReferral, ServiceCategory } from '../../services/interventionsService'
+import utils from '../../utils/utils'
+
+interface EndOfServiceReportFormPagePresenter {
+  text: {
+    title: string
+    pageNumber: string
+    numberOfPages: string
+  }
+}
+
+export default class EndOfServiceReportFormPresenter {
+  constructor(private readonly serviceCategory: ServiceCategory, private readonly referral: SentReferral) {}
+
+  private get numberOfDesiredOutcomes(): number {
+    return this.referral.referral.desiredOutcomesIds.length
+  }
+
+  private get title(): string {
+    return `${utils.convertToProperCase(this.serviceCategory.name)}: End of service report`
+  }
+
+  private get numberOfPages(): string {
```

Comment:
The latest designs don't seem to have page numbers on them, so we might be able to get rid of this in a subsequent PR.